static/css/restyle.less

```diff
@@ -0,0 +1,853 @@
+                border: none;
+                border-radius: 0;
+                box-shadow: 0 0 4px rgba(0, 0, 0, 0.4);
+                top: 25px;
+
+                /* Menu Panel trinagle */
+                &:after {
+                    content: "";
+                    display: block;
+                    position: absolute;
+                    top: -6px;
+                    z-index: 1;
+                    transform: rotate(45deg);
+                    background-color: #fff;
+                    width: 12px;
+                    height: 12px;
+                    box-shadow: 0 0 1px rgba(0, 0, 0, 0.4);
+                }
+            }
+
+            > a, &:hover > a {
+                background: none;
+                border: none;
+                box-shadow: none;
+            }
+
+            /* Menu Panel White block covering lower area of triangle and
+             * helping keep the
+             * mouse on hover */
+            &:hover:after {
+                content: "";
+                width: 200px;
+                z-index: 62;
+                height: 16px;
+                background: none;
+                background-color: #FFF;
+                position: absolute;
+            }
+        }
+    }
+}
+
+
+#aux-nav > ul > li {
+    > ul:after {
+        right: 10px;
+    }
+
+    &:hover:after {
+        top: 32px;
+        right: 1px;
+        height: 10px;
+    }
+}
+
+#site-nav > ul > li {
+    // arrow
+    > ul:after {
+        left: 10px;
+    }
+
+    &:hover:after {
+        top: 25px;
+        left: 1px;
+        height: 10px;
+    }
+}
+
+#site-nav #themes ul.two-col {
+    padding: 8px;
+    height: 270px;
+
+    div {
+        display: none;
+        top: 314px;
+        left: -1px;
+    }
+}
+
+/* Top Menu */
+/*seperator */
+#aux-nav > ul > li:not(:first-child):before {
+    ul {
+        li {
+            /* Other Applications Panel trinagle position*/
+            ul:after {
+                left: auto;
+                right: 15px;
+            }
+
+            /* Other Applications Menu Arrow */
+            a:after {
+                display: none;
+            }
+
+            /* Other Applications Panel white block position*/
+            &:hover:after {
+                top: 24px;
+                left: -65px;
+                width: 120px;
+            }
+
+            &not(:first-child):before {
+                background:linear-gradient(to bottom,
+                  transparent 0%,
+                  transparent 20%,
+                  #fff 20%,
+                  #fff 80%,
+                  transparent 80%
+                );
+            }
+        }
+    }
+}
+
+.header-search {
+    background: none;
+    border-radius: 0;
+    box-shadow: none;
+
+    input {
+        border-radius: 0;
+        box-shadow: none;
+        font: inherit;
+    }
+}
+
+button.search-button {
+    background-image: url("https://addons.cdn.mozilla.net/static/img/icons/go-arrow.png?ad8fa66"),
```

Comment:
Will `//` URLs work here? I guess there's no harm in always serving it over HTTPS, so ignore me if you like :smile: